src/app/icd/server/ICDMonitoringTable.cpp

```diff
@@ -140,7 +144,20 @@ CHIP_ERROR ICDMonitoringEntry::SetKey(ByteSpan keyData)
     Crypto::Symmetric128BitsKeyByteArray keyMaterial;
     memcpy(keyMaterial, keyData.data(), sizeof(Crypto::Symmetric128BitsKeyByteArray));
 
-    // TODO - Add function to set PSA key lifetime
+#ifdef CONFIG_CHIP_CRYPTO_PSA
```

Comment:
We can't use `CONFIG_CHIP_CRYPTO_PSA` here, as this is Kconfig and it does not exist for non-Zephyr based platforms. I think we could use `CHIP_CRYPTO_PSA` crypto define instead.

Reply:
Right, didn't notice `CONFIG_` prefix... but for the new approach it is not needed anymore, fortunately :)